_maps/map_files/stations/deltastation.dmm

```diff
@@ -78888,7 +78886,7 @@
 	dir = 8
 	},
 /obj/machinery/camera{
-	c_tag = "Research Lobby";
+	c_tag = "Power Transmission Laser";
 	dir = 1;
 	network = list("Research","SS13")
```

Comment:
Same here - can likely be narrowed to just "SS13"

Reply:
sorted the networks as well